java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -418,14 +418,16 @@ CompletableFuture<Map<GlideString, Map<GlideString, GlideString[][]>>> xreadBina
      * @param start Starting stream entry ID bound for range.
      *     <ul>
      *       <li>Use {@link IdBound#of} to specify a stream entry ID.
-     *       <li>Use {@link IdBound#ofExclusive} to specify an exclusive bounded stream entry ID.
+     *       <li>Since Redis 6.2.0, use {@link IdBound#ofExclusive} to specify an exclusive bounded
```

Comment:
```suggestion
     *       <li>Since server version 6.2.0, use {@link IdBound#ofExclusive} to specify an exclusive bounded
```
or
```suggestion
     *       <li>Since Valkey 6.2.0, use {@link IdBound#ofExclusive} to specify an exclusive bounded
```

Please apply to all